app/components/task_list_items/cil_liability_component.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module TaskListItems
+  class CilLiabilityComponent < TaskListItems::BaseComponent
+    def initialize(planning_application:)
+      @planning_application = planning_application
+    end
+
+    private
+
+    attr_reader :planning_application
+
+    delegate(:cil_liable, to: :planning_application)
+
+    def link_text
+      "CIL Liability"
```

Comment:
Should we not be extracting UI text to a locale file? Understand we don't want to spend time in a sprint to go back and change things but we probably want to be proactive about not making things worse.